src/commands/githubActions/bumpUpstream/index.ts

```diff
@@ -153,6 +157,24 @@ Compose - ${JSON.stringify(compose, null, 2)}
 
   const versionsToUpdate = Array.from(versionsToUpdateMap.values());
   manifest.upstreamVersion = getUpstreamVersionTag(versionsToUpdate);
+
+  const ethProvider = await tryEthProviders({
```

Comment:
What if you provide your own provider? it should be contemplated as the first opt

Reply:
Where would you provide it? The `CliGlobalOptions` the gaBumpUpstreamHandler` is receiving does not allow this parameter

Reply:
Consider adding it